pkgs/development/interpreters/octave/default.nix

```diff
@@ -1,4 +1,4 @@
-{ stdenv, fetchurl, gfortran, readline, ncurses, perl, flex, texinfo, qhull
```

Comment:
I think current default build does build info files — do you mean that it doesn't need `texinfo` for building them?